sw/SoC/examples/hello_world/src/main.c

```diff
@@ -4,8 +4,10 @@
 int main()
 { 
 
-  tinyIO_init();
+  uint32_t uart_base_address = 0x200000;
```

Comment:
This should be the same for both physical and virtual UART.
- unify virtual_uart_config.csv and config.csv with a common UART address (e.g. 0x10000)
- Tip: move up physical UART, swap with GPIO out